tower-discover/src/list.rs

```diff
@@ -14,6 +14,8 @@ where
     inner: Enumerate<T::IntoIter>,
 }
 
+impl<T: IntoIterator> Unpin for ServiceList<T> {}
```

Comment:
Probably not okay.

Reply:
Same here, there is no unsafe thus this is 100% safe.